src/components/SearchAutocomplete.js

```diff
@@ -8,46 +8,48 @@ import Typography from '@material-ui/core/Typography';
 import TextField from '@material-ui/core/TextField';
```

Comment:
I just trimmed down this file. A lot of things were never used.

Reply:
kk